source/includes/aws/_buckets.md

```diff
@@ -41,22 +85,36 @@ curl -X GET \
 
 Retrieve a list of all buckets from Amazon S3.
 
-| Attributes                        | &nbsp;                                                                                                                                                                                                                   |
+| Attributes | &nbsp; |
 |-----------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
-| `id`<br/>*string*                 | The ID of the bucket, which is of the form ":regionName/:bucketName".                                                                                                                                                                                                  |
-| `name`<br/>*string*               | The name of the bucket.                                                                                                                                                                                                |
-| `region`<br/>*string*     | The region the bucket exists in.                                                                                                                                                     |
-| `created`<br/>*string*            | The date the bucket was created.                                                                                                                                                                         |
-| `url`<br/>*string*       | The full endpoint url used to make api calls on the bucket.                                                                                                                                                                                                                                               
+| `id`<br/>*string*                 | The ID of the bucket, which is of the form `:regionName/:bucketName`.
+| `name`<br/>*string*               | The name of the bucket.
+| `region`<br/>*string*             | The region the bucket exists in.
+| `created`<br/>*string*            | The date the bucket was created.
+| `url`<br/>*string*                | The full endpoint URL used to make API calls on the bucket.
+
+
 <!-------------------- Create bucket -------------------->
 
 #### Create Bucket
 
 ```shell
 curl -X POST \
    -H "MC-Api-Key: your_api_key" \
-   "https://cloudmc_endpoint/rest/services/aws/test-env/buckets/"
+   "https://cloudmc_endpoint/rest/services/aws/test-env/buckets?operation=create"
```

Comment:
is operation=create required here?

Reply:
i think so because the operation != http verb

Reply:
let me check though

Reply:
you're right :O how does it know?

Reply:
because its a POST on a bucket entity 
POST = create
PUT = update
DELETE = delete
in general